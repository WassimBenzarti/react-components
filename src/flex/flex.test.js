import React from "react";
import Flex from "."
import Enzyme, {shallow} from "enzyme"

import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Flex', () => {
  it('should show the children', () => {
    expect(Flex).toBeTruthy()
    const text = "Hello world";
    let element = shallow(<Flex>{text}</Flex>);
    expect(element.text()).toEqual(text);
  })

  it("should set the style correctly", ()=>{
    let element = shallow(<Flex justifyContent="center" alignItems="center" row></Flex>);
    expect(element.html()).toMatchSnapshot();
  })

})
