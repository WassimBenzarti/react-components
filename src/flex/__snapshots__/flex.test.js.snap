// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex should set the style correctly 1`] = `"<div style=\\"display:flex;flex-direction:row;justify-content:center;align-items:center\\"></div>"`;
